### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mistrysiddh**
Simplification of installing process and configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,21 @@ Display a solid wifi logo as network signal for MagicMirror<sup>2</sup>
 -   [NPM Ping](https://www.npmjs.com/package/ping)
 
 ## Installation
-
-1. Clone this repo into `~/MagicMirror/modules` directory.
-2. Install dependencies `npm i`
-3. Configure your `~/MagicMirror/config/config.js`:
-
-   ```
+Open up your terminal and past the following code into it.
```
Comment (PoOwAa) [inline README.md]:
```suggestion
Open up your terminal and paste the following code into it.
```

File: README.md [hunk 2]
```diff
@@ -10,12 +10,21 @@ Display a solid wifi logo as network signal for MagicMirror<sup>2</sup>
...
+```bash
+cd ~/MagicMirror/modules/
+git clone https://github.com/PoOwAa/MMM-network-signal.git
+cd MMM-network-signal
+npm i
+```
+## Confirigation
```
Comment (PoOwAa) [inline README.md]:
```suggestion
## Configuration
```

**Review by PoOwAa — CHANGES_REQUESTED** [review]
Your changes make sense to me. I like it. Please correct the typos, after that, I can merge! :)

**Comment by mistrysiddh** [comment]
sorry for the spelling mistakes I sometime hate autocorrection

**Review by PoOwAa — APPROVED** [review]
Thanks for contributing! 👍🏻

**Comment by mistrysiddh** [comment]
> Thanks for contributing! 👍🏻

it's my pleasure brother

**Pull request merged**